src/components/panel/select-view/DisableSelectView.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Checkbox } from '@/components/ui/checkbox.tsx'
+
+
+interface Props {
+  updateCheckedValue: (value: boolean) => void
+}
+
+const DisableSelectView: FC<Props> = ({ updateCheckedValue }) => {
+  const isChecked = useRef<boolean>(false)
+  const { t } = useTranslation()
+
+  function handleCheckedChange(checked) {
+    isChecked.current = !!checked
+    updateCheckedValue(isChecked.current)
+  }
+
+  return (
+    <div className="flex items-center space-x-2">
+      <Checkbox id="do-not-ask-again" onCheckedChange={(checked) => handleCheckedChange(checked)} />
+      <label
```

Comment:
this label should be lighter than the title of the dialog, make it normal color or muted and make it regular font weight